core/tests/behavior/read_only.rs

```diff
@@ -122,6 +125,55 @@ pub async fn test_stat_not_exist(op: Operator) -> Result<()> {
     Ok(())
 }
 
+/// Stat with if_match should succeed, else get a ConditionNotMatch error.
+pub async fn test_stat_with_if_match(op: Operator) -> Result<()> {
+    let path = "normal_file";
```

Comment:
Maybe we should check `capability.stat_if_match` first?

Reply:
Nice catch.